_search-plugins/querqy/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Querqy plugin
+has_children: false
+redirect_from:
+  - /search-plugins/querqy/
+nav_order: 10
+---
+
+# Querqy plugin
+
+Querqy is a community plugin for query rewriting that helps to solve relevance issues, making search engines more precise regarding matching and scoring.
+
+## Querqy plugin installation
+
+**Querqy is currently only compatible with OpenSearch 1.3.1**
+
+The Querqy plugin code is located here: [querqy-opensearch](https://github.com/querqy/querqy-opensearch). To download the plugin code ZIP file, select the green "Code" button, then select "Download ZIP"
```

Comment:
Since this is a process and ordering matters, convert these steps into a numbered list.

Reply:
Converted to numbered list as per suggestion